src/main/webapp/app/exercises/shared/dashboards/tutor/exercise-assessment-dashboard.component.html

```diff
@@ -72,20 +68,18 @@
                     </div>
                     <div class="col-5">
                         <div *ngIf="exercise.isAtLeastInstructor">
-                            <a [routerLink]="submissionsLink">
-                                <span class="text-success">{{ numberOfSubmissions.inTime }}</span>
-                                <span *ngIf="numberOfSubmissions.late > 0"> - </span>
-                                <span *ngIf="numberOfSubmissions.late > 0" class="text-danger">{{ numberOfSubmissions.late }}</span>
-                            </a>
+                            <span class="text-success">{{ numberOfSubmissions.inTime }}</span>
+                            <span *ngIf="numberOfSubmissions.late > 0"> - </span>
+                            <span *ngIf="numberOfSubmissions.late > 0" class="text-danger">{{ numberOfSubmissions.late }}</span>
                             |
-                            <a [routerLink]="submissionsLink" [queryParams]="{ tutorId: tutor?.id }">{{ numberOfTutorAssessments }} ({{ tutorAssessmentPercentage }}%)</a>
+                            {{ numberOfTutorAssessments }} ({{ tutorAssessmentPercentage }}%)
                         </div>
                         <div *ngIf="!exercise.isAtLeastInstructor">
                             <span class="text-success">{{ numberOfSubmissions.inTime }}</span>
                             <span *ngIf="numberOfSubmissions.late > 0"> - </span>
                             <span *ngIf="numberOfSubmissions.late > 0" class="text-danger">{{ numberOfSubmissions.late }}</span>
                             |
-                            <a [routerLink]="submissionsLink" [queryParams]="{ tutorId: tutor?.id }">{{ numberOfTutorAssessments }} ({{ tutorAssessmentPercentage }}%)</a>
+                            {{ numberOfTutorAssessments }} ({{ tutorAssessmentPercentage }}%)
```

Comment:
This is now duplicated (the same code for instructors and tutors), can you merge that?